tests/functional/generate_pages.py

```diff
@@ -29,7 +29,7 @@
 
 HEADER = r"""// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
 
-import {basePage} from '../../{basePageFile}'
+import {basePage} from '../{basePageFile}'
```

Comment:
Although we should probably sort out the generation of the SummaryPage now it's a block; I updated to stop the script exploding on them but it produces an empty Summary for now.